Makefile

```diff
@@ -3,7 +3,6 @@ GO_REQUIRED_MIN_VERSION = 1.16
 # Include the library makefiles
 include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	golang.mk \
-	targets/openshift/bindata.mk \
```

Comment:
bindata per se might still be necessary for the operator, let's leave it

Reply:
Ok, then I'll create an empty dir with bindata. Otherwise the verify target fails.